mdjs-layout/doc/index.md

```diff
@@ -36,20 +29,61 @@ const demoContent = `
     </details>
   </div>
 `;
+
+// context object showing what Backlight generates based on the project meta info
+const demoContext = {};
 ```
 
 ```js preview-story
-export const demo = () => {
-  const configChanged = (ev) => {
-    const change = ev.detail;
-    if (change.key === 'color-scheme-dark-mode') {
-      console.table([change])
-      // Do any changes to sync with your DS dark-mode logic
+import '@divriots/dockit-core/mdjs-layout/load-all';
+import { breakpoints } from '@divriots/dockit-core/mdjs-layout';
+import { html } from 'lit';
+import { unsafeHTML } from 'lit/directives/unsafe-html.js';
+import { logoSvg } from './logo.svg.js';
+
+const docLayoutTemplate = (content, context) => html`
+  <style>
+    .logo {
+      color: #f59e0b;
     }
-  }
+    .topbar {
+      width: 100%;
+      justify-content: right;
+    }
+    @media screen and (min-width: ${breakpoints.xl}) {
+      .topbar {
+        justify-content: unset;
+      }
+    }
+    /* setup proper colors for the demo container */
+    /* it doesn't have to be the same as the doc shell/typography background */
+    .preview-story .story_padded {
+      background-color: #f3f4f6;
+    }
+    html.dark .preview-story .story_padded {
+      background-color: #1f2937;
+    }
+  </style>
+  <mdjs-layout
+    .context="${context}"

```

Comment:
this is the biggest thing I overlooked and only see this now when integrating it in the dockit-core... dockit-core so far doesn't know about Backlight internals like MDJS layout `context` and originally we were not planning to couple `mdjs-layout` component with Backlight... but sorry, I coupled it which means people will need to mimic the context structure if the want to use it outside of Backlight which is not truly good Open Source

I need to think how to best solve and if some abstraction would be better (e.g. `mdjs-layout` will have it's own structure for the navigation items and there will be a utility which will convert `context` to what `mdjs-layout` can consume)

Reply:
I don't really see this as an issue if the property is optional. It's essentially just passing the file tree so we can populate a sidebar menu right? So if you don't pass context, the essentials will still work?

Reply:
it wasn't, but not it will work correctly in such case, so I consider this resolved, thanks 👍